logger/default.go

```diff
@@ -0,0 +1,60 @@
+package logger
+
+import "io"
+
+type defaultLogger struct{}
```

Comment:
Can you rename this to something that makes it clearer that it's a no-op logger? For example `nopLogger`?